user-interface/pom.xml

```diff
@@ -122,6 +122,11 @@
       <artifactId>ehcache</artifactId>
       <classifier>jakarta</classifier>
     </dependency>
+    <dependency>
+      <groupId>javax.cache</groupId>
+      <artifactId>cache-api</artifactId>
+      <version>1.1.0</version>
+    </dependency>
```

Comment:
NIT: Is this dependency really necessary? I don't see it used in the changed code segment. 
If it's necessary why not use the "newest" [1.1.1 ](https://mvnrepository.com/artifact/javax.cache/cache-api/1.1.1)version?